examples/webgl_geometry_text_earcut.html

```diff
@@ -47,38 +47,54 @@
 		<!-- replace built-in triangulation with Earcut -->
 		<script src="js/libs/earcut.js"></script>
 		<script>
+			
+			function removeDupEndPts(points) {
```

Comment:
This method would be better as a private function defined in the scope of `triangulateShape()`, as it is in the other examples, don't you think?